cli/script/command-parser.ts

```diff
@@ -261,11 +261,10 @@ var argv = yargs.usage(USAGE_PREFIX + " <command>")
     .command("logout", "Log out of the current session", (yargs: yargs.Argv) => {
         isValidCommandCategory = true;
         isValidCommand = true;
-        yargs.usage(USAGE_PREFIX + " logout [--local]")
+        yargs.usage(USAGE_PREFIX + " logout")
             .demand(/*count*/ 1, /*max*/ 1)  // Require exactly one non-option argument.
-            .example("logout", "Log out and also remove the access key used for the current session")
-            .example("logout --local", "Log out but allow the use of the same access key for future logins")
-            .option("local", { demand: false, description: "Whether to delete the current session's access key on the server", type: "boolean" });
+            .example("logout", "Log out and end your session");
+            // The 'logout --local' option is obsoleted
```

Comment:
I'd prefer not having "x is obsoleted" comments littered throughout the codebase (here and everywhere else).  Commit/release descriptions can summarize this.  Otherwise, there will be additional work at some future point in time to remove these comments.  Comments should exist to clarify intent of existing code when necessary, not to explain what happened to code of yesteryear.

If you feel strongly about communicating that a feature is obsoleted, you could add that back in the console output when --local is passed.  I don't think this is necessary if release notes communicate it instead.